packages/components/pie-form-label/package.json

```diff
@@ -33,6 +33,8 @@
   "devDependencies": {
     "@custom-elements-manifest/analyzer": "0.9.0",
     "@justeattakeaway/pie-components-config": "0.11.0",
+    "@justeattakeaway/pie-input": "^0.11.0",
+    "@justeattakeaway/pie-switch": "^0.26.0",
```

Comment:
Apologies if I'm missing the point, but are these dependencies needed?

Reply:
Ah I see, it's for the tests

Reply:
Sadly Playwright will crash without having those as deps. Perhaps there is any other way of solving this?